spot_rl_experiments/configs/ros_topic_names.yaml

```diff
@@ -14,6 +14,7 @@ IMAGE_SCALE: "/image_scale"
 ROBOT_STATE: "/robot_state"
 TEXT_TO_SPEECH: "/text_to_speech"
 
+ODOM_TOPIC: "/spot_odom"
```

Comment:
@jimmytyyang @KavitShah1998 Does this new topic name follow Spot naming conventions?

Reply:
I don't think we do have any naming convention for spot's ros frames. It would be a good idea to add one.

Reply:
Let's rename to `/robot_odom`

Reply:
addressed